docs/survey/graph/README.md

```diff
@@ -1,18 +1,5 @@
-The search for gathering a team of experts who are expected to collaboratively work towards accomplishing a given
-project successfully is referred to as Team Formation, a problem that has historically been solved in a variety of
-ways including manually in a time-consuming and bias-filled manner, and, algorithmically within such disciplines as
-social sciences, management, operations technology, and so forth. In the present effort, while providing a taxonomy to
-distinguish between search-based versus learning-based approaches, we survey the graph-based Team Formation studies
-from the search-based category, motivated as they comprise the mainstream. We present a unifying and vetted overview
-of the various definitions of the notions in this realm, scrutinize assumptions and benchmarks, and identify shortfalls.
-We start by drawing upon initial efforts and approaches to the problem of Team Formation to lay the conceptual
-foundations and set forth the necessary notions for a more grounded view of this realm. Next, with the help of a
-specifically designed set of notations, we provide a detailed view of the graph-based Team Formation approaches based
-on the objective functions they aim to optimize. In doing so, we lay out who builds on whom and how algorithms have
-evolved to solve previous works’ drawbacks. Additionally, we categorize different evaluation schemes and elaborate on
-different metrics and the insights that can be drawn from each. Referring to the evaluation schemas and metrics, we
-compare different works and propose future directions.
+## A Survey of Subgraph Optimization for Expert Team Formation
+Expert Team Formation is the search for gathering a team of experts who are expected to collaboratively work towards accomplishing a given project, a problem that has historically been solved in a variety of ways, including manually in a time-consuming and bias-filled manner, and algorithmically within disciplines like social sciences and management. In the present effort, while providing a taxonomy to distinguish between search-based versus learning-based approaches, we survey graph-based studies from the search-based category, motivated as they comprise the mainstream. We present a unifying and vetted overview of the various definitions in this realm, scrutinize assumptions, and identify shortfalls. We start by reviewing initial approaches to the Expert Team Formation problem to lay the conceptual foundations and set forth the necessary notions for a more grounded view of this realm. Next, we provide a detailed view of graph-based Expert Team Formation approaches based on the objective functions they optimize. We lay out who builds on whom and how algorithms have evolved to solve the drawbacks of previous works. Further, we categorize evaluation schemas and elaborate on metrics and insights that can be drawn from each. Referring to the evaluation schemas and metrics, we compare works and propose future directions.
```

Comment:
Hi @mahdis-saeedi 
Thank you for the change. 
Would you please add sth that shows this repo is about an accepted/published survey and holds complementary information/resource/artifacts, link to paper and journal, ... 
That is when a general user ends up here, knows where this location is.